Documentation/Storage-Configuration/Shared-Filesystem-CephFS/filesystem-storage.md

```diff
@@ -220,6 +220,135 @@ If the Rook cluster has more than one filesystem and the application pod is sche
 
 Once you have pushed an image to the registry (see the [instructions](https://github.com/kubernetes/kubernetes/tree/release-1.9/cluster/addons/registry) to expose and use the kube-registry), verify that kube-registry is using the filesystem that was configured above by mounting the shared filesystem in the toolbox pod. See the [Direct Filesystem](../../Troubleshooting/direct-tools.md#shared-filesystem-tools) topic for more details.
 
+## Consume the Shared Filesystem across namespaces
+
+A PVC that you create using the `rook-cephfs` storage class can be shared between different Pods simultaneously, either read-write or read-only, but is restricted to a single namespace (a PVC is a namespace-scoped resource, so you cannot use it in another one).
```

Comment:
`storage class` to `Storageclass`

Reply:
If it is in a sentence I would leave it as "storage class"

Reply:
Kept as it was.